Makefile

```diff
@@ -121,3 +121,4 @@ bin/etcdctl:
 .PHONY: clean
 clean:
 	rm -rf bin/*
+	rm -f templates/config/*.cfg
```

Comment:
This was to clean the generated files and should have been removed when I backed out the templates dir.